kratos/processes/flux_based_redistance_process.h

```diff
@@ -0,0 +1,649 @@
+//    |  /           |
```

Comment:
There is quite a lot of commented code and formatting issues in this file. I'd acknowledge a bit of clean up for the review. Thanks.